PriceQuoteIos/Utils/APIHandler.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  APIHandler.swift
+//  PriceQuoteIos
+//
+//  Created by Livia Vasconcelos on 27/09/20.
+//  Copyright © 2020 Livia Vasconcelos. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+
+class APIHandler {
+    public static func handleAPIResponse<T: BaseResponse> (data: DataResponse<T>,
+                                                           completion: @escaping (BaseCallback<T>) -> Void) {
+        switch(data.result) {
+        case .success(let response):
+            switch response.status {
+            case 201:
```

Comment:
You could use the whole 200 http code range here, since all of them is related to success:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status

Reply:
Removed.